src/test/features/createCampaign.feature

```diff
@@ -0,0 +1,24 @@
+@campaign
+Feature: Report
+
+Background: 
+ Given User passes the authorization on "Auth Page"
+
+ Scenario: Generate a new report
+        Given User logins as a "admin"
+        And User is in "Dashboard" page
+        When User clicks "Create Campaign Button" on "Dashboard Page"
+        And User clicks "Direct Advertiser Company Type" on "Campaign Wizard Page"
+        And User enters "Company Name Input" as "YM" and selects "YM Company" from list on "Campaign Wizard Page"
```

Comment:
Single responsibility? Does the business user need to know that you're performing two actions? You can describe it in one action.